proto/scripts/swagger.py

```diff
@@ -50,9 +50,12 @@ def clean(path: str, patch: str) -> None:
         elif key == "protobufNullValue":
             value["title"] = "NullValue"
 
-        # Clean up titles.
+        # Clean up titles. Title is used in documentation.
         if "title" not in value:
             value["title"] = "".join(capitalize(k) for k in key.split(sep="v1"))
+        elif value["title"].startswith(SERVICE_NAME):
```

Comment:
the way titles are generated seems to be different for enums vs messages defined in proto.  in our current setup, this check precisely targets the enums we want but we could maybe add a regex to skip titles like `Determined abc` or assert that this is not the case